gui.go

```diff
@@ -76,6 +76,9 @@ type Gui struct {
 	testCounter int // used for testing synchronization
 	testNotify  chan struct{}
 
+	// The position of the mouse
+	mouseX, mouseY int
```

Comment:
What about settings the default values for these to -1 so people can detect is the mouse is at 0,0 without it triggering on app start?